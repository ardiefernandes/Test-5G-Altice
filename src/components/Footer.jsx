import React from "react";
import Altice_Logo from "./../assets/altice_logo_white_footer.png";

const Footer = () => {
  return (
    <footer className="app-footer">
      <img alt="alice logo" src={Altice_Logo} />
    </footer>
  );
};

export default Footer;
