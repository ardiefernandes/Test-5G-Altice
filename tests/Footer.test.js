import React from "react";
import renderer from "react-test-renderer";
import Footer from "../src/components/Footer";

it("correctly renders the footer", () => {
  const component = renderer.create(<Footer />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
