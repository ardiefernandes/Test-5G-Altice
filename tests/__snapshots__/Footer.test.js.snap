// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders the footer 1`] = `
<footer
  className="app-footer"
>
  <img
    alt="alice logo"
    src="altice_logo_white_footer.png"
  />
</footer>
`;
